build.gradle

```diff
@@ -5,4 +5,86 @@ plugins {
 task buildDockerImages() {
     dependsOn(':TrafficCapture:dockerSolution:buildDockerImages')
     dependsOn(':RFS:buildDockerImages')
+}
+
+subprojects {
```

Comment:
This worries me a bit, do all of our gradle commands we reference and use in our scripts now need to change since this is at the top level now?

Reply:
This adds the below configuration to all subprojects, which only the javadoc compile to all test tasks.

I've excluded RFS from the maven publishing step.

No changes need to occur to how the existing gradle commands are invoked. The previous PR did change the commands based on the working directory (e.g `./gradlew dockerSolution:composeUp` -> `../gradlew dockerSolution:composeUp`)

Reply:
To Tanner's point, gradlew moved to the top level a couple changes ago.  I (& some others) updated the path to gradlew.  I'm pretty sure that with this block being lifted up, it just means that more subprojects will be configured properly.